frontend/pages/[station_category_slug]/[station_slug].tsx

```diff
@@ -113,6 +118,8 @@ export default function StationPage({
         <meta name="twitter:card" content="summary" />
         <meta name="keywords" content={seo?.keywords} />
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
+        {hostname !== "www.radio-crestin.com" &&
```

Comment:
It seems that `direct.radio-crestin.com` does not have this `noindex` tag:
`view-source:http://direct.radio-crestin.com/radio/rve-cluj`